zapetw/unsupported_linux.go

```diff
@@ -0,0 +1,28 @@
+package zapetw
```

Comment:
Can you please explain what is this file used? I see we do not support Linux OS now and you have same functions but write noETWEvent support in this file. Do we need this file? Or the filename should be core_linux.go and maybe put some comments here?